vowpalwabbit/core/src/reductions/cb/cb_adf.cc

```diff
@@ -234,6 +235,7 @@ void VW::reductions::cb_adf::learn(learner& base, VW::multi_ex& ec_seq)
   if (VW::test_cb_adf_sequence(ec_seq) != nullptr)
   {
     _offset = ec_seq[0]->ft_offset;
+    _offset_index = _offset / _all->weights.stride();
```

Comment:
this should be something like _offset / self.increment (self being the learner that owns this cb_adf data instance) but we don't have access to that info at this point

Reply:
@jackgerrits